client/src/components/home/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+
+import propTypes from 'prop-types';
+import Slider from './slider/slider';
+import Button from '../utils/Button';
+import Pop from '../utils/PopUp';
+
+import './style.css';
+
+class Home extends Component {
+  state = {
+    logoutComp: null,
+    callWaiter: null,
+  };
+
+  componentWillUnmount() {
+    this.setState({ logoutComp: null, callWaiter: null });
```

Comment:
I don't think you need to setState here.  When `componentWillUnmount` is called, the component is basically about to be destroyed, so `this.state` will disappear automatically.

"You should not call setState() in componentWillUnmount() because the component will never be re-rendered. Once a component instance is unmounted, it will never be mounted again."

https://reactjs.org/docs/react-component.html#componentwillunmount